src/koa-middleware.js

```diff
@@ -93,6 +93,12 @@ class KoaMiddleware {
             }
         });
 
+        // If proper route is not found, send an undefined route
+        // The caller is responsible for setting a default "N/A" route in this case
+        if (!properRoute) {
+            return undefined;
```

Comment:
In the `_handleResponse` method, this will set the route label to 'N/A' though the `||` operator. Let me know if there is a cleaner solution